graylog2-web-interface/src/components/configurations/UrlWhiteListForm.jsx

```diff
@@ -0,0 +1,213 @@
+        } else if (type === regex && value.trim().length > 0) {
+          const promise = ToolsStore.testRegexValidity(value);
+          promise.then((result) => {
+            const res = result.is_valid ? { valid: true } : { valid: false };
+            _updateValidationError(idx, type, name, res, value);
+          });
+        } else {
+          const res = { valid: false };
+          _updateValidationError(idx, type, name, res, value);
+        }
+        break;
+      default:
+        break;
+    }
+  };
+
+  const _onInputChange = (event: SyntheticInputEvent<EventTarget>, idx: number, type: string) => {
+    _validate(event.target.name, idx, type, FormUtils.getValueFromInput(event.target));
+  };
+
+  const _onUpdateType = (idx: number, type: string) => {
+    const stateUpdate = ObjectUtils.clone(config);
+    _validate('value', idx, type, stateUpdate.entries[idx].value);
+  };
+  const _getSummary = () => {
+    return (config.entries.map((url, idx) => {
+      return (
+      // eslint-disable-next-line react/no-array-index-key
+        <tr key={url.id}>
+          <td>{idx + 1}</td>
+          <td>
+            <Input type="text"
+                   id={`title-input${idx}`}
+                   ref={(elem) => { inputs[`title${idx}`] = elem; }}
+                   help={validationState.errors[idx] && validationState.errors[idx].title && !validationState.errors[idx].title.valid ? 'required' : null}
+                   name="title"
+                   bsStyle={validationState.errors[idx] && validationState.errors[idx].title && !validationState.errors[idx].title.valid ? 'error' : null}
+                   onChange={event => _onInputChange(event, idx, url.type)}
+                   defaultValue={url.title}
+                   required />
+          </td>
+          <td>
+            <Input type="text"
+                   id={`value-input${idx}`}
+                   ref={(elem) => { inputs[`value${idx}`] = elem; }}
+                   help={validationState.errors[idx] && validationState.errors[idx].value && !validationState.errors[idx].value.valid ? 'Not valid' : null}
+                   name="value"
+                   bsStyle={validationState.errors[idx] && validationState.errors[idx].value && !validationState.errors[idx].value.valid ? 'error' : null}
+                   onChange={event => _onInputChange(event, idx, url.type)}
+                   defaultValue={url.value}
+                   required />
+          </td>
+          <td>
+            <Input id={`url-input-type-${idx}`}
+                   required
+                   autoFocus>
+              <Select placeholder="Select Cache Type"
+                      clearable={false}
+                      options={options}
+                      matchProp="label"
+                      onChange={option => _onUpdateType(idx, option)}
+                      value={url.type} />
+            </Input>
+          </td>
+          <td>
+            <span className="">
```

Comment:
As far as I can see we don't need this `span` element.

Reply:
Ok, i cleaned all unnecessary tag or properties

Reply:
Fixed